Automattic/jetpack
# Settings UI: add writing tab settings, optimistic toggles, fix state tree update

#### Changes proposed in this Pull Request:
- introduce settings for Writing tab
- make instantaneous optimistic toggles, both for modules or for an option that relies on a module (module is activated and option is set)
- `updateFormStateModuleOption()` now accepts a third parameter to state if the module should be deactivated
- reveal options only after a module is enabled
- make FormFieldset able to display a help icon
- add all needed help links
- add inline link to toggle fancy atd options
- update some module descriptions to use them in JS
- Site Verification fields now accepts the entire meta tag

#### Testing instructions:
* build and check Jetpack > Settings. Make sure that:
- toggles work instantly
- Post by Email regeneration works and display the address
- all help links works
- site verification works both with full meta tags like `<meta name="google-site-verification" content="1234abcd-5678_efgh" />` and just the code like `1234abcd-5678_efgh`
